### ISSUE TITLE
Changing folder name of page caused pages to be removed

### ISSUE THREAD

**Issue by marshmn** [getgrav/grav-plugin-admin#1660]
I have some pages with a structure something like this:

```
/ 
/tournament
/tournament/pre-register
```

Each of the above pages are actually modular pages with sub-page children - but the structure is essentially as above.

I wanted to add another level into my paths, so what I decided I wanted to do was to change the structure to be like this:

```
/
/tournament/2019
/tournament/2019/pre-register
```

I wasn't quite certain of the best way to go about changing this, but I thought I'd try just changing the 'Directory Name' field on the /tournament page. So I changed that from 'tournament' -> 'tournament/2019' and saved.

Doing this caused an error to be displayed: 
"Creating directory failed for /var/www/html/src/user/pages/02.tournament/2019/modular.md"

However, **it also caused all my pages from /tournament and below to be removed**. An empty file named '02.tournament' seems to have been created in the place where the '02.tournament' directory once stood.

Note that folder prefix numbering was in operation - I'm not sure whether or not that is a relevant factor here.

**Comment by rhukster** [comment]
Ok, you certainly can't do things the way you did it, it should probably throw an error and shouldn't delete things.  However, what you should do, is create an empty folder under the `/tournament` level called `2019`.  Then after that, set the parent of the `pre-register` page to that folder you just created.

**Comment by rhukster** [comment]
I just tested it and it actually worked the way you did it. I would still follow my advice and do it that way though.

**Comment by marshmn** [comment]
I realised (afterwards) that they way I'd done it was clearly not the way I should have done it - which is fair enough.

However, as you said, the worrying thing was that files were deleted, which definitely shouldn't happen. (not a problem for me in this case since I was working on my local copy and could restore from version control - but others might not be so 'lucky'...)

Before raising the issue I did re-test it a couple of times and the same thing happened again. So maybe there is something more particular about my setup that is related to it. I'm not in a position to test again right now, but I will give it another try when I can and report back.
